install_config/upgrading/manual_upgrades.adoc

```diff
@@ -1613,142 +1613,145 @@ common issues:
-deployments.  In an {product-title} environment the most suitable candidate for
-blue-green deployments are the Nodes.  Uptime is most important for these
-workloads and the additional complexity of blue-green deployments can be
-justified.  Often the main hurdle to overcome is having the excess capacity to
-facilitate such an approach.
+there are always trade-offs to be made. Not all environments have the same
+uptime requirements or the resources to properly perform blue-green deployments.
+In an {product-title} environment the most suitable candidate for blue-green
+deployments are the Nodes. Uptime is most important for these workloads and the
+additional complexity of blue-green deployments can be justified. Often the main
+challenge is having the excess capacity to facilitate such an approach.
 
 ifdef::openshift-enterprise[]
 Another lesser challenge is that the administrator will have to temporarily
 share the entitlements between the blue-green deployments or provide access to
-the installation content by means of a system such as Red Hat Satellite.  This
+the installation content by means of a system such as Red Hat Satellite. This
 can be accomplished by sharing the consumer ID from the previous host.
 
 On the old host:
 ----
-subscription-manager identity | grep system
+# subscription-manager identity | grep system
 system identity: 6699375b-06db-48c4-941e-689efd6ce3aa
 ----
 
 On the new host:
 ----
-subscription-manager register --consumerid=6699375b-06db-48c4-941e-689efd6ce3aa
+# subscription-manager register --consumerid=6699375b-06db-48c4-941e-689efd6ce3aa
 ----
 
 [IMPORTANT]
 ====
-When the deployment has been declared successful, it is important that the old
-host be unregistered with `*subscription-manager clean*` to prevent the
-environment from being out of compliance.
+When the deployment is declared successful, it is important that the old host be
+unregistered with `*subscription-manager clean*` to prevent the environment from
+being out of compliance.
 ====
 
 endif::openshift-enterprise[]
 
+[[manual-upgrades-preparing-for-upgrade]]
 === Preparing for Upgrade
 
-xref:#upgrading-masters[After the master and etcd servers have been upgraded]
-you will need to ensure that your current production nodes are labeled either
-blue or green.  In this example the current installation will be blue and the
-new environment will be green.
+xref:#upgrading-masters[After the master and etcd servers have been upgraded],
+you need to ensure that your current production nodes are labeled either blue or
+green. In this example, the current installation will be blue and the new
+environment will be green.
 
 ----
-oc label --all nodes color=blue
+$ oc label --all nodes color=blue
 ----
 
-There is quite a bit of flexibility here.  Perhaps there was only some subset of
-nodes that required the uptime guarantees of a blue-green deployment.  In that
+There is quite a bit of flexibility here. Perhaps there is only some subset of
```

Comment:
* "There is ... flexibility here." is noisy to my ears.
* "Perhaps ... deployment.  In that case, BAR..." can probably be condensed into "In the case of ..., BAR" (or something like that).